server/resp/src/test/java/org/infinispan/server/resp/RespSingleNodeTest.java

```diff
@@ -1231,6 +1231,7 @@ public void testPFADD() {
       // TODO: Verify cardinality ISPN-14676
 
       assertWrongType(() -> redis.set("plain", "string"), () -> redis.pfadd("plain", "el1"));
+      // assertWrongType(() -> redis.pfadd("data", "e1"), () -> redis.get("data"));
```

Comment:
You can write to an HLL and retrieve it with a GET command, and it works if the respCache uses Protostream. Still, for other serialization mechanisms, the command fails with an ERRWRONGTYPE.